configure.ac

```diff
@@ -1143,6 +1115,7 @@ else
       MINIUPNPC_CPPFLAGS="-DSTATICLIB -DMINIUPNP_STATICLIB"
     fi
   else
+    use_upnp=no
```

Comment:
Why change this?

Reply:
without this the final value of `use_upnp` was `auto`, the default value. But it started happening only after removing libc related changes. I didn't investigate further, just fixed it.